pkg/sharings/sharings.go

```diff
@@ -198,6 +199,66 @@ func findSharingRecipient(db couchdb.Database, sharingID, clientID string) (*Sha
 	return sharing, sRec, nil
 }
 
+// sendDoc sends a JSON document to a recipient
+func sendDoc(docType, id string, doc *couchdb.JSONDoc, recStatus *RecipientStatus) error {
+	// Get the recipient info
+	token := recStatus.AccessToken
+	rec := recStatus.recipient
+	domain, err := rec.ExtractDomain()
+	if err != nil {
+		return err
+	}
+	path := fmt.Sprintf("/data/%s/%s", docType, id)
+	// A new doc will be created on the recipient side
+	delete(doc.M, "_id")
+	delete(doc.M, "_rev")
+	body, err := request.WriteJSON(doc.M)
+	if err != nil {
+		return err
+	}
+
+	client := new(http.Client)
+	_, err = request.Req(&request.Options{
+		Domain: domain,
+		Client: client,
```

Comment:
I think you only need to give a value to Client if you want to do something special (like it's done in client package tests). Otherwise, the default value of Client is http.DefaultClient which should work. Confirmation @jinroh ?